osf/models/user.py

```diff
@@ -931,9 +931,10 @@ def draft_registrations_active(self):
         """
         Active draft registrations attached to a user (user is a contributor)
         """
+
         return self.draft_registrations.filter(
-            models.Q(registered_node__isnull=True) |
-            models.Q(registered_node__is_deleted=True),
+            (models.Q(registered_node=None) | models.Q(registered_node__deleted__isnull=False)),
```

Comment:
I think I asked this before, but why the change from `registered_node__isnull=True` to `registered_node=None`?

Reply:
IIRC It just seemed simpler and more concise.

Reply:
`__isnull=True` technically has different behavior than `=None`, so please revert to that